src/routes/dashboard/pet-list/CreatePet.svelte

```diff
@@ -0,0 +1,155 @@
+<script lang="ts">
+  import FileUpload from '../../../assets/svg/file-upload.svg'
+  import { Gender, PetType } from '../../../services/api/pets/contracts'
+  import type { CreatePetDto } from '../../../services/api/pets/contracts'
+  import { format, getTime } from 'date-fns'
+
+  export let onCancel: () => void // PR issue - discuss what to do with empty arrow functions (tslint errors). Wanted to assing one here and one below.
+  export let onCreate: (newPet: CreatePetDto) => void
```

Comment:
`pet` should be enough. Spare your keyboard any additional keystrokes ;)

```suggestion
  export let onCreate: (pet: CreatePetDto) => void
```

Reply:
Will do